### PR TITLE
Remove f lines

### PR DESCRIPTION
**Pull request by Voros2**
This is to remove the F lines in the credits file, as fraggle mentioned in https://github.com/freedoom/freedoom/issues/299

### CODE REVIEW

**Comment by chungy** [comment]
This isn't really an appropriate edit since you're replacing them with a redundant blank line, and you added a long (unbroken!) comment at the top that will hurt the machine readability of the file. I'll just run a sed script ...

File: CREDITS [hunk 1]
```diff
@@ -1,305 +1,307 @@
-F: cph
+This is a list of all the contributors that have submitted their work to the Freedoom project, since the first release in 2003, during the development phase from 2001 and onwards. It's a long list.
```
Comment (fragglet) [inline CREDITS]: Please remove this line; this file is supposed to be machine readable.

File: CREDITS [hunk 2]
```diff
@@ -1,305 +1,307 @@
...
 N: Colin Phipps
 S: cph
 E: cph@cph.demon.co.uk
 W: http://www.cph.demon.co.uk/
 D: Binary lumps (playpal, colormap etc)
 
-F: julian
+
```
Comment (fragglet) [inline CREDITS]: Please remove duplicate empty lines; it's sufficient to just have a single empty line between each contributor.

**Comment by fragglet** [comment]
Never mind; I see this has already been done. Thanks anyway.

**Pull request closed without merging**